dir.targets

```diff
@@ -44,9 +44,11 @@
           AfterTargets="_RestoreBuildToolsWrapper">
     <MSBuild Projects="$(RepoRoot)\targets\DeployDependencies.proj"
              Targets="DeployDependencies"
-             BuildInParallel="true"
-             Condition="'$(OsEnvironment)'=='Windows_NT'"
+             Properties="DeploymentPath=$(DeploymentDir)"
```

Comment:
OK, after looking further you're only creating two calls to this: one to assemble the "deployment" directory and one for the test directory.  That kills my "just put it in a `.targets`" idea.

So let's just be careful about the distinct-outputs problem.  I'll look closely at `DeployDependencies.proj`.